source/articles/2015/save-money-and-be-happier-by-updating-your-gems-every-monday-morning.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Save Money and Be Happier by Updating Your Gems Every Monday Morning
+description: An approach to avoiding the technical debt of gem drift
+
+author: Adarsh Pandit
+category: writing
+date: 2015-11-12
+logo: ruby_on_rails_logo.png
+published: true
+tags: [ruby on rails, update gems, versions, technical debt, upgrade gems]
+---
+
+As a newcomer to Rails,
+understanding Gems and the Gemfile
+is fairly straightforward:
+You include a gem that you need
+and you get it included everywhere.
+
+We don't often think about it,
+but that last bit is important:
+we are _literally including_
```

Comment:
Yeah it's a bit overwrought. I changed to "literally `include`ing" to be less intense and more specific in 1157b9d